### DISCUSSION TITLE
SDRPlay error cannot start device

### DISCUSSION THREAD

**Discussion by jeagh** [jvde-github/AIS-catcher#321] [Q&A]
I built AIS-catcher a month or so ago and it worked well with an RTL-SDR. I've since bought an SDRPlay RSPdx-r2 receiver  and I've rebuilt AIS-catcher.

The receiver is detected.

 AIS-catcher -l
Found 1 device(s):
0: SDRPLAY, UNKNOWN, SN: 2402094A70

But when I enter AIS-catcher -v 10 -N 8100

Searching for device with SN 2402094A70.
Warning: sample rate 230K upsampled to 288K.
TCP Server: start thread at port 8100
HTML Server running at port 8100
Error: SDRPLAY: cannot start device
TCP Server: thread ending.

Can anyone suggest why my receiver cannot be started?

By the way I have AIS-catcher installed on Xubuntu.

**Comment by jvde-github** [comment]
Hello, I suspect the issue is with the sample rate. Can you try setting it explicitly by adding -s 3072K ?

I changed the default sample rate in the source but I think it is good practice to experiment with the best gain settings and sample rate for your device.

Hope this works, Jasper
